akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/BasicClusterExampleTest.java

```diff
@@ -69,7 +69,7 @@ public void clusterApiExample() {
 
       // #prepare
       PrepareForFullClusterShutdown msg =
-          PrepareForFullClusterShutdown.prepareForFullClusterShutdown();
+          PrepareForFullClusterShutdown$.MODULE$.prepareForFullClusterShutdown();
```

Comment:
Ugh, that's no nice Java API.

Reply:
yeah, we might want to lobby for those static forwarders to be generated again in the scala compiler - I haven't looked into why they're not yet.